tests/foreman/api/test_environment.py

```diff
@@ -24,44 +24,54 @@
 from requests.exceptions import HTTPError
 
 from robottelo.api.utils import one_to_many_names
-from robottelo.datafactory import filtered_datapoint
+from robottelo.datafactory import invalid_environments_list
 from robottelo.datafactory import invalid_names_list
-from robottelo.test import APITestCase
+from robottelo.datafactory import parametrized
+from robottelo.datafactory import valid_environments_list
 
 
-@filtered_datapoint
-def valid_data_list():
-    """Return a list of various kinds of valid strings for Environment entity"""
-    return [gen_string('alpha'), gen_string('numeric'), gen_string('alphanumeric')]
+@pytest.fixture(scope='module')
+def module_org():
```

Comment:
There is a `module_org` fixture available, you should use that here instead of re-defining it.

Its defined in `pytest_fixtures/api_fixtures.py`, but no need to import it, pytest will resolve it automatically.

Reply:
Same for the fixtures below - check out what is available in `api_fixtures.py`, and just use their names in test function parameters.

Reply:
ACK, changed